### PR TITLE
Frontend baseline

### PR DESCRIPTION
**Pull request by stephenneale**
PR will add a basic Django project and application called DimSpace.

### CODE REVIEW

File: as-developers/frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+## Requirements
+
+- python3
+	$ sudo apt install python3
+- virtualenv
+	$ sudo apt install python3-virtualenv
+	$ virtualenv venv
+	$ chmod 777 venv/bin/activate
+- django
+	$ pip install python
```
Comment (matt-lebl) [inline as-developers/frontend/README.md]: Was this supposed to be django? :p
```suggestion
	$ pip install django
```

File: as-developers/frontend/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Run
+
+Enter virtualenv
+	$ . venv/bin/activate
```
Comment (matt-lebl) [inline as-developers/frontend/README.md]: Hmmm, I'm having trouble activating the virtual environment. I don't have a lot of experience with it, but shouldn't there be some `venv/activate` script someplace?

File: as-developers/frontend/README.md (hunk 1 above)
Comment (stephenneale) [reply as-developers/frontend/README.md]: Correct, I'll fix this :)

File: as-developers/frontend/README.md (hunk 2 above)
Comment (stephenneale) [reply as-developers/frontend/README.md]: Ah, there is a setup command that I left out. I'll add this in a revision.

Comment (stephenneale) [reply as-developers/frontend/README.md]: I've updated the instructions, let me know if you have any trouble.

File: as-developers/frontend/dimspace-main/dimspace/settings.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-5pn63tjsp*b5tz1wnoac0q8e!u05@5jm0l&#qhb=^h#aq3z*5_'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = ["159.223.198.2"]
```
Comment (matt-lebl) [inline as-developers/frontend/dimspace-main/dimspace/settings.py]:
```suggestion
ALLOWED_HOSTS = ["159.223.198.2", "localhost"]
```

**Review by matt-lebl — APPROVED** [review]
The site looks great! It took a little fiddling to get it to work on my machine, though. I added `localhost` to the allowed hosts in `settings.py` (otherwise i got an invalid host error), and additionally navigating to the root at `localhost:8000` gave me a 404. Only when I manually went to `localhost:8000/dimspace` did it work.

Great work! With a couple fixes I think this is A+.

**Comment by matt-lebl** [comment]
Also, mockups are 💯.

**Comment by matt-lebl** [comment]
🎉

**Pull request merged**